mkosi/__init__.py

```diff
@@ -1371,23 +1557,42 @@ def make_image(state: MkosiState, skip: Sequence[str] = [], split: bool = False)
             definitions.mkdir()
             bootdir = state.root / "usr/lib/systemd/boot/efi"
 
+            # If grub for BIOS is installed, let's add a BIOS boot partition onto which we can install grub.
+            bios = (state.config.bootable != ConfigFeature.disabled and
+                    want_grub_bios(state) and
+                    any(gen_kernel_images(state)))
+
+            if bios:
+                (definitions / "05-bios.conf").write_text(
+                    textwrap.dedent(
+                        """\
+                        [Partition]
+                        Type=2168614864496E6F744E656564454649
```

Comment:
Since they live in the same file, couldn't this be
```suggestion
                        Type={Partition.GRUB_BOOT_PARTITION_UUID}
```